startup.py

```diff
@@ -89,10 +90,17 @@ def prepare_launch(self, exec_path, args, file_to_open=None):
         # Signals which engine instance from the environment is going to be used.
         required_env["SHOTGUN_ENGINE"] = self.engine_name
 
-        # Set the bootstrap location in the environment variable that will be used by the Unreal Shotgun startup script
+        # Set the bootstrap location in the environment variable that will be used by the Unreal Shotgun plugin
+        # startup script
         bootstrap_script = os.path.join(self.disk_location, "plugins", "basic", "bootstrap.py")
+        # From UE5, the UE_SHOTGRID_BOOTSTRAP environment variable is supported.
+        # We still need to support UE_SHOTGUN_BOOTSTRAP for previous UE versions.
         required_env["UE_SHOTGUN_BOOTSTRAP"] = bootstrap_script
-
+        required_env["UE_SHOTGRID_BOOTSTRAP"] = bootstrap_script
+        # UE5 Python ignores PYTHONPATH and only uses UE_PYTHONPATH
+        # blindly copy what was set so modules (e.g. SG TK core) are found
+        # when bootstrapping.
+        required_env["UE_PYTHONPATH"] = os.environ.get("PYTHONPATH") or ""
```

Comment:
For the `PYTHONPATH` vs `UE_ PYTHONPATH` problem, there is nothing clever than I can do since the environment is not set by tk-unreal, so I blindly copy over everything, which ruins the purpose of having a new env. variable, but I would argue that having this new env. variable in the first place is may be not so wise....